packages/mongo/tests/hooks/direct.js

```
@@ -0,0 +1,81 @@
+createTest('direct_collection_test_stringid1', { connection: null })
+createTest(null, { connection: null })
+
+if (Meteor.isServer) {
+  Tinytest.addAsync(
+    'direct - Meteor.users.direct.insert should return _id, not an object',
+    async function (test) {
+      await Meteor.users.removeAsync('directinserttestid')
+
+      const result = await Meteor.users.direct.insertAsync({
+        _id: 'directinserttestid',
+        test: 1
```

ESLint

[error] Insert `,`

(prettier/prettier)